File: Services/CategoryServices/DrivingSchoolApp.Services.CategoryServices.CCategoryAPI/DataAccess/Entities/Truck.cs
namespace DrivingSchoolApp.Services.CategoryServices.CCategoryAPI.DataAccess.Entities
{
    public class Truck : Base
    {
        public int TruckNumber { get; set; }
        public string Type { get; set; }
        public string Model { get; set; }
        public int Year { get; set; }
		public string Fuel { get; set; }
		public int Power { get; set; }
		public string Transmission { get; set; }
		public int CylinderCapacity { get; set; }
		public string Color { get; set; }
		public string Photo { get; set; }
		public int AvailableTruck { get; set; }
		public IEnumerable<TruckPerCity> TruckPerCities { get; set; } = new List<TruckPerCity>();
	}
}

File: Frontend/DrivingSchoolWeb/src/app/categories/category-c/models/truck.ts
import {Base} from 'src/app/categories/shared/models/base';

export interface Truck extends Base{
  truckNumber:number,
  type:string,
  model:string,
  year:number,
  fuel:string,
  power:number,
  transmission:string,
  cylinderCapacity:number,
  color:string,
  photo:string,
  availableTruck:number
}
